### ISSUE TITLE
about being applied the batch-norm/2d

### ISSUE THREAD

**Issue by khryang** [junyanz/pytorch-CycleGAN-and-pix2pix#83]
I attempted to run the cycle-gan model
I'm wondering that the batch-norm applies each images
I mean the batch_size in base_option.py is "1",
so the normalization applies each one image, is it right??

actually, I'm very thanks for sharing your research and code. 
It's very helpful to me :-)

**Comment by junyanz** [comment]
@khryang  Thanks for your nice words. By default, the code should use the instancenorm. When batchSize=1, instancenorm=batchnorm, and, you are right, the normalization applies to each image.

**Comment by khryang** [comment]
@junyanz  oh! I see, thank you for nice and fast answering

**Comment by hazirbas** [comment]
Hi @junyanz and @khryang ,
Since batchnorm mode is not set to evaluation, batches are normalized by their mean and variance.
